python/ray/tests/kuberay/test_autoscaling_config.py

```diff
@@ -336,6 +422,61 @@ def _fetch_ray_cr_from_k8s(self) -> Dict[str, Any]:
             assert out == {"ok-key": "ok-value"}
 
 
+TPU_PARAM_ARGS = ",".join(
+    [
+        "ray_cr_in",
+        "expected_num_tpus",
+    ]
+)
+TPU_TEST_DATA = (
+    []
+    if platform.system() == "Windows"
```

Comment:
No need to have the if else here since we already have `skipif` down below.

Reply:
Done in 9d186c5